netCDF4/utils.py

```diff
@@ -190,12 +190,10 @@ def _StartCountStride(elem, shape, dimensions=None, grp=None, datashape=None,\
     else:   # Convert single index to sequence
         elem = [elem]
     
-    hasEllipsis=False
-    for e in elem:
-        if type(e)==type(Ellipsis):
-            if hasEllipsis:
-                raise IndexError("At most one ellipsis allowed in a slicing expression")
-            hasEllipsis=True
+    # ensure there is at most 1 ellipse
+    if sum((1 for e in elem if type(e) == type(Ellipsis))) > 1:
```

Comment:
You can omit the extra `()` inside sum.

Also, there is only one `Ellipsis` object (it's a singleton), so `e is Ellipsis` would suffice.

Reply:
You also don't need the extra inner set of parenthesis.